lib/travis/build/script/matlab.rb

```diff
@@ -26,16 +25,20 @@ def setup
           super
 
           # Echo support notice
-          sh.echo MATLAB_NOTICE, ansi: :green
-          
-          # Execute helper script to install runtime dependencies
-          sh.raw "wget -qO- --retry-connrefused #{MATLAB_DEPS_LOCATION}" \
-                 ' | sudo -E bash -s -- $TRAVIS_MATLAB_VERSION'
-
-          # Invoke the ephemeral MATLAB installer that will make a MATLAB available
-          # on the system PATH
-          sh.raw "wget -qO- --retry-connrefused #{MATLAB_INSTALLER_LOCATION}" \
-                 ' | sudo -E bash -s -- --release $TRAVIS_MATLAB_VERSION'
+          MATLAB_NOTICE.each do |message|
+            sh.echo message, ansi: :green
+          end
+
+          sh.fold 'Setup MATLAB' do
```

Comment:
Have you seen any precedent for having a space in the fold title? The one I see seem to either separate words with underscores, dashes, or periods and keep it all lowercase. Like:

`haxe-install`
`rustup-install`
`crystal_install`
`nix.install`

Reply:
Hmm I'm seeing it used in different ways across the different languages. Julia, for example, has `Julia-install` for its fold 🤔 
That being said, we could go with something more in line with the other languages (because perhaps they didn't use spaces for a reason) 🤔

`matlab-setup`?
`MATLAB-setup`?

Reply:
I vote `matlab-install` or `matlab-setup`.